RELEASE_NOTES.md

```diff
@@ -15,5 +15,5 @@
 - [KOGITO-4894](https://issues.redhat.com/browse/KOGITO-4894) - Remove `image` default value in KogitoRuntime CR (problem in OCP console)
 - [KOGITO-4781](https://issues.redhat.com/browse/KOGITO-4781) - Deployment constantly updates when deployed with custom probes
 - [KOGITO-4993](https://issues.redhat.com/browse/KOGITO-4993) - Operator crash when data-index is created
-
+- [KOGITO-5005](https://issues.redhat.com/browse/KOGITO-5005) - Operator print irrelevant metadata in logs
```

Comment:
We still don't have a script which reset the RELEASE NOTES on master once branched ...
Could you remove all the other issues in there ? ...

Reply:
done